testing/tests/DevExpress.viz.gauges/barGauge_new.tests.js

```diff
@@ -8,6 +8,23 @@ import loadingIndicatorModule from 'viz/core/loading_indicator';
 
 import 'viz/gauges/bar_gauge';
 
+const environment = {
+    beforeEach() {
+        this.renderer = new vizMocks.Renderer();
+
+        sinon.stub(rendererModule, 'Renderer', () => {
+            return this.renderer;
+        });
+    },
+    afterEach() {
+        rendererModule.Renderer.restore();
+    },
+
+    createGauge(options) {
+        return $('<div>').appendTo($('#qunit-fixture')).dxBarGauge(options).dxBarGauge('instance');
```

Comment:
It's just easier to read
```suggestion
        return $('<div>')
            .appendTo($('#qunit-fixture'))
            .dxBarGauge(options)
            .dxBarGauge('instance');
```